.idea/integreat-locales.iml

```diff
@@ -6,7 +6,7 @@
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/native-locales" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/tools" isTestSource="false" />
-      <excludeFolder url="file://$MODULE_DIR$/external-jobs" />
+      <sourceFolder url="file://$MODULE_DIR$/external-jobs" type="java-resource" />
```

Comment:
is `java-resource` necessary/correct here?

Reply:
I removed this. The `java-resource` type is set if you mark a directory as `resource directory`.